tests/Main.hs

```diff
@@ -152,6 +152,21 @@ main = Hspec.hspec $ do
                   (defaultPublicKeyCredentialCreationOptions pkCredential)
                   pkCredential
         registerResult `shouldSatisfy` isRight
+  describe "Apple register" $
+    it "tests whether the fixed apple register has a valid attestation" $
+      do
+        pkCredential <-
+          either (error . show) id . JS.decodeCreatedPublicKeyCredential Fido2.allSupportedFormats
+            <$> decodeFile
+              "tests/responses/attestation/05-apple.json"
+        let registerResult = do
+              toEither $
+                Fido2.verifyAttestationResponse
+                  (M.Origin "https://6cc3c9e7967a.ngrok.io")
+                  (rpIdHash "6cc3c9e7967a.ngrok.io")
+                  (defaultPublicKeyCredentialCreationOptions pkCredential)
+                  pkCredential
+        registerResult `shouldSatisfy` isRight
```

Comment:
I'm just realizing this now, but we should really check the result against what we expect. As in something like
```haskell
let expectedResult = Right M.CreatedPublicKeyCredential { ... }
registerResult `shouldBe` expectedResult
```

The value for `expectedResult` should be verified against the source of the JSON